app/src/main/java/org/fossasia/openevent/app/core/track/list/TracksFragment.java

```diff
@@ -95,6 +95,7 @@ protected int getTitle() {
     public void onStop() {
         super.onStop();
         refreshLayout.setOnRefreshListener(null);
+        initialized = false;
```

Comment:
If this is to be reinitialized again and again, then remove the boolean and check altogether